Description: Correct pod NAME issue in LedgerSMB/Template/TTI18N.pm
 Missing the dash from the pod NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472640&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Template/TTI18N.pm
+++ b/LedgerSMB/Template/TTI18N.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::Template::TTI18N  Template Toolkit i18n support functions
+LedgerSMB::Template::TTI18N - Template Toolkit i18n support functions
 
 =head1 SYNOPSIS
 
